deepfence_worker/ingesters/vulnerabilites.go

```diff
@@ -73,7 +73,7 @@ func CVEsToMaps(ms []ingestersUtil.Vulnerability) ([]map[string]interface{}, err
 			"rule":    utils.ToMap(rule),
 			"data":    utils.ToMap(data),
 			"scan_id": v.ScanID,
-			"node_id": data.CveCausedByPackage + rule.CveID,
+			"node_id": data.CveCausedByPackagePath + data.CveCausedByPackage + rule.CveID,
```

Comment:
```suggestion
			"node_id": strings.Join([]string{data.CveCausedByPackagePath, data.CveCausedByPackage, rule.CveID}, "_")
```
this will make **node_id** more readable